src/providers/azure/mod.rs

```diff
@@ -46,6 +46,9 @@ Content-Transfer-Encoding: base64
 
 ";
 
+// this is a known working wireserver endpoint within Azure
```

Comment:
Nit: if you make this a docstring (`///`) it will appear in rustdocs.

Also, can you please add a reference to https://blogs.msdn.microsoft.com/mast/2015/05/18/what-is-the-ip-address-168-63-129-16/?